lang/python/python-libsemanage/Makefile

```diff
@@ -0,0 +1,87 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+SRC_NAME:=libsemanage
+PKG_NAME:=python-$(SRC_NAME)
+PKG_VERSION:=3.1
+PKG_RELEASE:=1
```

Comment:
missing `PKG_LICENSE`, `PKG_LICENSE_FILES` and `PKG_CPE_ID` (if assigned)

Reply:
Fixed.